rfc/rfc017-distributed-network-grpc-v2.md

```diff
@@ -54,26 +54,116 @@ As specified by RFC015, the node MUST authenticate the peer's node DID as follow
 1. Resolve peer's node DID to its corresponding DID document.
 2. Assert that one of the `NutsComm` endpoint's host matches (one of) the `dNSName` SANs in the peer's TLS client certificate.
 
-## 4. Set reconciliation protocol
+## 4. Conversations
 
-One of the problems in a distributed network is how to make sure every node has processed all transactions.
-When two nodes haven't processed the same set of transactions, the second problem is how to efficiently synchronize the missing transactions.
-The higher the efficiency, the more transactions the entire network can process.
-The protocol described here is a compromise on efficiency and fairness. It's able to quickly determine a set of missing transactions, but also doesn't favour any node.
+Certain messages in the protocol are a response to a certain request. To make sure a response corresponds to a certain request, a `conversationID` is added to the request. Response type messages MUST add the same `conversationID` in the response.
+The `conversationID` is scoped to a connection.
+A node is free to choose the form of a `conversationID`.
```

Comment:
I'd say it MUST be unique? Or unique enough, so that for at least the validity period there are no duplicates?

To avoid implementations using a hardcoded string (or counting 0-9 and then rolling over).